src/shared/get-options.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Get options from window._activityPubOptions.
+ *
+ * @param {string|undefined} key - Optional key to retrieve specific option.
+ * @return {*|boolean} Option value if found, false if not found, or entire options object if no key provided.
+ */
+const getOptions = ( key ) => {
```

Comment:
How would you feel about `getOption` to reflect only the one option that's being passed?